src/main/java/nl/tudelft/semgroup4/collision/DefaultCollisionHandler.java

```diff
@@ -230,4 +248,34 @@ public void run() {
         Game.LOGGER.log(LogSeverity.DEBUG, "Collision", "Player picked up a utility");
         util.activate(game.getCurLevel());
     };
+    
+    private final CollisionHandler<HorMovingWall, AbstractWall> horWallWallHandler = 
+            (game, movingWall, abstractWall) -> {
+        int offset = 5;        
+        
+        if (movingWall.getLocX() < abstractWall.getLocX() 
```

Comment:
What does this block do? Can you maybe contain it in a method somewhere?

The off-set checking seems necessary, we have been doing it since the start of our project, but it would be very cool if we could somehow fix this problem, or handle it at a lower level ( I'm thinking about the intersection method of SemRectangle. Maybe it can return something to specify where the object is compared to the other object. Otherwise some static check method for SemRectangle?).

< organisation, layout, interface, logic